deploy/helm/sumologic/conf/logs/collector/common/filelog_receiver.yaml

```diff
@@ -99,6 +99,7 @@ filelog/containers:
       output: strip-trailing-newline
       source_identifier: attributes["log.file.path"]
       type: recombine
+      max_unmatched_batch_size: 1
```

Comment:
is this related to the bug preventing us from upgrading from `0.92.0-sumo-0`?

Reply:
Yes, related issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31653
and documentation: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/recombine.md#max_unmatched_batch_size-set-to-1